readme.md

```diff
@@ -136,23 +148,23 @@ $ npm -v
 * **Question:** What is `path` and where does it come from?
 
 * **Your Answer:**
-
+path is an environment variable and returns the logical path of the current directory
```

Comment:
In this case, `path` is a module that is provided for you via Node by default. It has _methods_ that will give you access to the current directory.